docs/sdk-and-tools/indexer.md

```diff
@@ -0,0 +1,139 @@
+---
+id: indexer
+title: Elasticindexer service
+---
+
+[comment]: # (mx-abstract)
+
+## Overview
+
+A MultiversX observer node can send messages over `WebSocket` to an elastic indexer service, which will process and index the data in an Elasticsearch database.
+
+The GitHub repository for the `elasticindexer` service can be found [here](https://github.com/multiversx/mx-chain-es-indexer-go).
+
+[comment]: # (mx-context-auto)
+
+## Architectural Overview
+
+The observer node in the network will be connected to `elasticindexer` service.
+
+:::important
+Set up one observer for each shard in order to handle all the data in the chain.
+:::
+
+![img](/technology/indexer.png)
+
+In the figure above:
+- The observer nodes will send WebSocket messages to the `elasticindexer` service.
+- The `elasticindexer` service will receive and process the WebSocket messages and index data in an Elasticsearch cluster.
+
+[comment]: # (mx-context-auto)
+
+## Set up observer and elasticindexer
+
+[comment]: # (mx-context-auto)
+
+### Observer Client
+
+On the observer side, there is a WebSocket host that will send messages to the elasticindexer service.
+
+In the observer node's configuration directory, `external.toml` config file can be configured
+to enable host driver config. The config file can be found
+[here](https://github.com/multiversx/mx-chain-go/blob/master/cmd/node/config/external.toml).
```

Comment:
this link does not work (yet):
use this one?
https://github.com/multiversx/mx-chain-go/blob/rc/v1.6.0/cmd/node/config/external.toml

Reply:
changed the link